src/styles/toast/toastContainer.scss

```
@@ -1,61 +1,59 @@
 /* stylelint-disable */
```

현재 작성하신 내용으로 보면 
```suggestion
/* stylelint-disable */
```
이건 삭제해도 좋을거같아요

---

stylelint class pattern때문에 적용해야 할 것 같아요